cyral/data_source_cyral_integration_logs_test.go

```diff
@@ -0,0 +1,124 @@
+package cyral
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const (
+	integrationLogsDataSourceName = "integration-log-datasource"
+)
+
+func TestAccLogsIntegrationDataSource(t *testing.T) {
+	// Using vars from resource test
+	testConfigElk, testFuncElk := setupLogsTest(initialLogsConfigElk)
+	testUpdateConfigElk, testUpdateFuncElk := setupLogsTest(updatedLogsConfigElk)
+
+	testConfigCloudWatch, testFuncCloudWatch := setupLogsTest(initialLogsConfigCloudWatch)
+	testUpdateConfigCloudWatch, testUpdateFuncCloudWatch := setupLogsTest(updatedLogsConfigCloudWatch)
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
Please make sure that these are not flaky tests and are independent of the existing resources in a CP. You can use the tests for the existing data sources as a reference.

Reply:
Thanks, I improved the datasource tests in here: [1c83973](https://github.com/cyralinc/terraform-provider-cyral/pull/395/commits/1c839735700362291d95554a13a9e1995f341452)